public/sass/base/variables.scss

```diff
@@ -1,7 +1,8 @@
 $exercism-red: #d81d4e;
 $exercism-dark-red: #6e1f2d;
 $exercism-medium-red: #770428;
-$exercism-charcoal: #333;
-$exercism-medium-plus-gray: #777;
+$exercism-charcoal: #333333;
+$exercism-medium-plus-gray: #777777;
```

Comment:
I like how you're making the colour format consistent using the 6 character version.